### PR TITLE
[QE] Test cases for edit mappings

### PR DESCRIPTION
**Pull request by kpunwatk**
1. Rhv provider edit mapping test case for both network and storage mappings
2. Vmware provider edit mapping test case for both network and storage mappings
new file:   pkg/qe-tests/cypress/integration/tests/Rhv/edit_mapping_test.ts
new file:   pkg/qe-tests/cypress/integration/tests/vmware/edit_mapping_test.ts

### CODE REVIEW

**Comment by kpunwatk** [comment]
@ibragins Please review this PR, Thanks in advance !

File: pkg/qe-tests/cypress/integration/tests/Rhv/edit_mapping_test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { MappingNetwork } from '../../models/mappingNetwork';
+import { editNetworkMapping, editStorageMapping, testData } from './config_separate_mapping_rhv';
+import { login } from '../../../utils/utils';
+import { MappingStorage } from '../../models/mappingStorage';
+
+describe('Creating CRUD operation', () => {
```
Comment (sshveta) [inline pkg/qe-tests/cypress/integration/tests/Rhv/edit_mapping_test.ts:6]: CRUD operations include creating .
So just "CRUD Operations" is fine.



      describe('CRUD operations', () => {

File: pkg/qe-tests/cypress/integration/tests/vmware/edit_mapping_test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { MappingNetwork } from '../../models/mappingNetwork';
+import { editNetworkMapping, editStorageMapping, testData } from './config_separate_mapping';
+import { login } from '../../../utils/utils';
+import { MappingStorage } from '../../models/mappingStorage';
+
+describe('Creating CRUD operation', () => {
```
Comment (sshveta) [inline pkg/qe-tests/cypress/integration/tests/vmware/edit_mapping_test.ts:6]: Same as above.

**Review by sshveta — DISMISSED** [review]

File: pkg/qe-tests/cypress/integration/tests/Rhv/edit_mapping_test.ts (hunk 1 above)
Comment (kpunwatk) [reply pkg/qe-tests/cypress/integration/tests/Rhv/edit_mapping_test.ts]: Thanks a lot, @sshveta for the suggestion. I've amended those changes and pushed.
Would you please review it again? Thanks in advance !

**Review by ibragins — APPROVED** [review]
LGTM

**Comment by yaacov** [comment]
closing as new development is moving to:
https://github.com/kubev2v/forklift-console-plugin

**Pull request closed without merging**